src/admin-views/tribe-options-tickets.php

```diff
@@ -76,6 +76,14 @@
 		'can_be_empty'    => true,
 		'validation_type' => 'options_multi',
 	],
+	'event_tickets_uninstall'   => [
+		'type'            => 'checkbox_bool',
+		'label'           => esc_html__( 'Remove all Event Tickets data upon uninstall?', 'event-tickets' ),
+		'tooltip'         => esc_html__( 'If checked, all Event Tickets data will be removed from the database when the plugin is uninstalled.', 'event-tickets' ),
```

Comment:
We may need to review the text eventually since we won't be uninstalling _all_ data initially.

Reply:
Fair point. I figured the goal would be to uninstall all the data eventually, so it would be easier to add verbiage to that effect now.